Algebra.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+namespace CalcClassLibrary
+{
+    public class Algebra
+    {
+        public int Add(int a, int b)
+        { return a + b; }
+        public int Substract(int a, int b)
+        { return a - b; }
+        public int Multiply(int a, int b)
+        { return a * b; }
+        public float Divide(int a, int b)
+        { return a / b; }
```

Comment:
This should not take in any values. Please create internal properties and use them once you construct the object.